### ISSUE TITLE
Generated lineups are not in sequence by projection

### ISSUE THREAD

**Issue by utahman** [DimaKudosh/pydfs-lineup-optimizer#125]
I use:
lineup_generator = optimizer.optimize_lineups(lineups)
for lineup in lineup_generator:
   ...
And noticed that generated lineups are not in sequence by projected points.
It might look like this:
270.234
269.782
270.145
269.401

What might be the reason for that?
On one hand, its not difficult to sort them after all of them generated. But could it be an indication of a bug? Although the number of lineups I will need already given in the parameter 'lineups' I might stop the 'for' loop earlier and end up with not the best set of lineups.
Or is it internal thing in the Pulp solver? Maybe it runs several threads and sometimes returns worse lineup first and better lineup after?

**Comment by kpm1988** [comment]
Are you using randomness in your lineup generator?  When I have randomness turned off the lineups are in sequence by projected points and the top lineup matches what I would get with a simple, excel-based solver.

**Comment by utahman** [comment]
No randomness.
I also just figured that I can't sort generated lineups because the method above is used for late swap, so the sequence is important, otherwise contest entries will not match and such lineups might be rejected.

**Comment by DimaKudosh** [comment]
Optimize_lineups method returns lineups in the same order as list of passed lineups. This method takes each lineup from list, lock players with started games and optimize lineup like in optimize method. So projection for lineup depends on how many players have running games(locked). It's impossible to get the most optimal lineup before regenerating all the lineups. So if projections ordering is important you should get all lineups and after that sort it.

**Comment by utahman** [comment]
Yes, but in this specific case there is no locked players. All games are in the future. Which means that every new generated lineup must be worse than previous (or equal).

**Comment by DimaKudosh** [comment]
In this case all lineups should be ordered, maybe it's a bug with detection of players with running games. You can check players that counted as players with running games   with this code: `players = [player for player in optimizer if player.is_game_started]`

**Comment by utahman** [comment]
It seems the problem is in the way the game start time compared to start time:
Latest code has this:

time_now = datetime.now().replace(tzinfo=timezone(get_timezone()))

While I think it must be like that:

time_now = datetime.now(starts_at.tzinfo).replace(tzinfo=timezone('US/Eastern'))
